src/core/exec-invoke.c

```diff
@@ -3337,31 +3337,39 @@ static int apply_working_directory(
                 const char *home,
                 int *exit_status) {
```

Comment:
If you touch this, maybe remove `params` from signature? It's not used in the function.

Reply:
It is used, there's a line `if (params->flags & EXEC_APPLY_CHROOT)` in it, which uses it.

Reply:
Oops, sorry for the noise